public/Board.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
+import { sleepNow } from "./Helpers.js";
+import CardsDeck from "./CardsDeck.js";
+import Player from "./Player.js";
+
+export default class Board {
+  constructor() {
+    this.deck = this.cardGeneration(); //Generates deck of cards
+    this.mixedDeck = this.mix(this.deck); // Mixes deck of cards
+    this.players = [];
+  }
+
+  cardGeneration() {
+    const cardsArrGen = [];
+    const clubs = ["clubs", "diamonds", "hearts", "spades"];
+    const cards = ["As", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K"];
+
+    const cardsClubs = clubs.length;
+    const differentCards = cards.length;
+
+    for (let i = 0; i < cardsClubs; i++) {
+      for (let j = 0; j < differentCards; j++) {
+        const card = new CardsDeck(clubs[i], cards[j]);
+        cardsArrGen.push(card);
+      }
+    }
+    return cardsArrGen;
+  }
+
+  mix(arr) {
```

Comment:
A better name for this would be `shuffle`
```suggestion
  shuffle(arr) {
```